gatsby-node.js

```diff
@@ -0,0 +1,26 @@
+const path = require(`path`);
+
+// exports.createPage({
```

Comment:
if you will not use this please remove it

Reply:
Done!